R/mainTests.r

```diff
@@ -1200,8 +1200,7 @@ eda_combinations <- function(data) {
       unique() %>%
       filter(!is.na(.)) %>%
       filter(english_devolved_area_code != "") %>%
-      filter(english_devolved_area_code != ":") %>%
-      filter(english_devolved_area_code != "z") %>%
+      filter(!(english_devolved_area_code %in% Symbol)) %>%
```

Comment:
I don't think we want to filter out all symbols. Something like 'low' we'd never expect in the geography columns. Could either limit it to fewer symbols, or drop this and add Not available and it's symbol as a name/code combo in the lookup file so that it's caught there?

Reply:
This repeats across the other geography checks which do the same

Reply:
The more I think about it the more I think we should add the Not available / x combination to the lookup csv's and then stop filtering out codes here. These tests were written before we added the lookups and filtering out by code opens it up to using an incorrect name with a code. This is linked to this card on our trello board - https://trello.com/c/hxjJfrj6/2270-z-with-any-name-passes-the-screener-add-more-checks-to-lock-it-down-like